packages/core/src/Chip/ChipContent/ChipContent.tsx

```diff
@@ -49,36 +52,53 @@ const ChipContent: React.FC<InferProps<typeof propTypes>> = ({
   bridgeLabel = undefined,
   /* eslint-disable @typescript-eslint/naming-convention */
   BridgeIcon = undefined,
+  topLabel = undefined,
+  borderRadius = undefined,
+  justifyContent = 'center',
   ...props
 }) => (
   <ChipContentWrapper
     data-testid='chipContentWrapper'
     hasChildren={Boolean(children)}
+    hasTopLabel={Boolean(topLabel)}
+    borderRadiusOverride={borderRadius}
     disabled={disabled}
     selected={selected}
     size={size}
+    justifyContent={justifyContent}
     {...props}
   >
     {children}
-    {Boolean(image) && <ImageWrapper disabled={disabled}>{image}</ImageWrapper>}
-    {Boolean(Icon) && <Icon title={IconTitle} size='20px' />}
-    {Boolean(label) && (
-      <Text regular ml={Boolean(Icon) || Boolean(image) ? 2 : 0}>
-        {label}
-      </Text>
-    )}
-    {Boolean(facet) && (
-      <Text regular ml={1}>
-        {facet}
-      </Text>
-    )}
-    {Boolean(bridgeLabel) && <BridgeIcon title='to' size='16px' ml={2} />}
-    {Boolean(bridgeLabel) && (
-      <Text regular ml={2}>
-        {bridgeLabel}
-      </Text>
-    )}
-    {Boolean(action?.Icon) && action.title && <action.Icon title={action.title} size='16px' ml={2} />}
+    <Flex alignItems='center' height='100%'>
+      {Boolean(image) && <ImageWrapper disabled={disabled}>{image}</ImageWrapper>}
+      {Boolean(Icon) && <Icon title={IconTitle} size='20px' />}
+      <Box>
+        {Boolean(topLabel) && (
+          <Text regular ml={2} mb={1} fontWeight='bold'>
+            {topLabel}
+          </Text>
+        )}
+        <Flex>
```

Comment:
Using <Flex> here changes the display property to 'flex' vs the original `inline-flex` assigned in `<ChipContentWrapper>`. Is that ok? Same goes for line 72.

Reply:
Good callout. Everything was looking good on regression though, Chromatic tests showed all the existing components looked the same